### PR TITLE
[ar] Translate ar index to arabic

### PR DESCRIPTION
**Pull request by mboukhalfa**
This PR adds the minimal translation related to the file _index.md and splitting the old https://github.com/kubernetes/website/pull/44902 to smaller PRs

Part of https://github.com/kubernetes/website/issues/37546 and https://github.com/kubernetes/website/issues/44682

### CODE REVIEW

**Review by seifrajhi — COMMENTED** [review]
LGTM for Arabic

File: content/ar/_index.html
Comment (adowair) [inline content/ar/_index.html:13]:
```suggestion
[كوبيرنيتيس]({{< relref "/docs/concepts/overview/" >}})، يُعرف أيضًا باسم ك8س، هو نظام مفتوح المصدر لتنسيق وتكييف وإدارة التطبيقات المعبأة في حاويات.
```

File: content/ar/_index.html
Comment (adowair) [inline content/ar/_index.html:16]:
```suggestion
يقوم بتجميع الحاويات التي تشكل تطبيقاً في وحدات منطقية لتسهيل الإدارة والاكتشاف. يستند كوبيرنيتيس على [15 عامًا من الخبرة في إدارة أعباء عمل الإنتاج في جوجل](http://queue.acm.org/detail.cfm?id=2898444)، إلى جانب أفضل الأفكار والممارسات من المجتمع.
```

File: content/ar/_index.html
Comment (adowair) [inline content/ar/_index.html:22]:
```suggestion
تم تصميم كوبيرنيتيس وفقًا لنفس المبادئ التي تسمح لجوجل بتشغيل مليارات الحاويات أسبوعياً، ويمكنه التوسع دون الحاجة لزيادة فريق العمليات لديك.
```

File: content/ar/_index.html
Comment (adowair) [inline content/ar/_index.html:28]:
```suggestion
سواءً كنت تُجري إختباراً محلياً أو تدير مؤسسة عالمية، فإن مرونة كوبيرنيتيس تتكيف معك لتقديم تطبيقاتك دون إنقطاع وبسهولة بغض النظر عن مدى تعقيد حاجتك.
```

**Review by adowair — APPROVED** [review]
Looks great, thank you @mboukhalfa. Just some minor nits


/lgtm

**Comment by adowair** [comment]
/lgtm

/hold

Feel free to remove the hold to merge as is or with the nits addressed. Thanks!

**Comment by mboukhalfa** [comment]
> Feel free to remove the hold to merge as is or with the nits addressed. Thanks!

Comment addressed and the lgtm label removed after pushing can you relgtm ? 
/hold cancel

**Comment by adowair** [comment]
/lgtm

Thank you!

File: content/ar/_index.html
Comment (adowair) [inline content/ar/_index.html:16]:
```suggestion
يقوم بتجميع الحاويات التي تشكل معاً تطبيقاً في وحدات منطقية لتسهيل الإدارة والاكتشاف. يستند كوبيرنيتيس إلى [15 عاما من الخبرة في إدارة أعباء عمل الإنتاج في جوجل](http://queue.acm.org/detail.cfm?id=2898444)، إلى جانب أفضل الأفكار والممارسات من المجتمع.
```

File: content/ar/_index.html
Comment (adowair) [inline content/ar/_index.html:22]:
```suggestion
تم تصميم كوبيرنيتيس وفقا لنفس المبادئ التي تسمح لجوجل بتشغيل مليارات الحاويات أسبوعياً، ويمكنه التوسع دون الحاجة لزيادة فريق العمليات لديك.
```

File: content/ar/_index.html
Comment (adowair) [inline content/ar/_index.html:28]:
```suggestion
سواءً كنت تُجري إختباراً محلياً أو تدير مؤسسةً عالميةً، فإن مرونة كوبيرنيتيس تتكيف معك لتقديم تطبيقاتك دون إنقطاع وبسهولة بغض النظر عن مدى تعقيد حاجتك.
```

File: content/ar/_index.html
Comment (adowair) [inline content/ar/_index.html:37]:
```suggestion
لتحميل كوبيرنيتيس قم بزيارة صفحة [التحميل](/releases/download/).
```

**Review by adowair — COMMENTED** [review]
Some improvement suggestions for future revision

**Pull request merged**
